pootle/apps/pootle_store/models.py

```diff
@@ -1274,6 +1276,23 @@ def live(self):
         """Filters non-obsolete stores."""
         return self.filter(obsolete=False)
 
+    def create(self, *args, **kwargs):
```

Comment:
The same concern as for ProjectManager re using StoreManager.create() for filetypes logic.